test/jdk/java/lang/StrictMath/FdlibmTranslit.java

```diff
@@ -86,6 +86,10 @@ public static double log1p(double x) {
         return Log1p.compute(x);
     }
 
+    public static double exp(double x) {
```

Comment:
Perhaps leave out of this PR any `FdlibmTranslit` changes and later include `ExhaustingTest` changes in each separate PR for the functions being ported?

Reply:
Okay; I'll roll the addition of the exp method into the PR for the hyperbolics since exp is used in the hyperbolics implementation.